<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PAKU-BLR-01" ignored="0" name="AnnotationTransformer" tests="7" failures="1" timestamp="12 Sep 2022 11:55:34 GMT" time="0.127" errors="0">
  <testcase name="test1" time="0.018" classname="RetryAnalyzerDemo.MyTestCase1"/>
  <testcase name="test2" time="0.004" classname="RetryAnalyzerDemo.MyTestCase1"/>
  <testcase name="test3" time="0.005" classname="RetryAnalyzerDemo.MyTestCase1"/>
  <testcase name="test5" time="0.007" classname="RetryAnalyzerDemo.MyTestCase1">
    <skipped/>
  </testcase> <!-- test5 -->
  <testcase name="test5" time="0.002" classname="RetryAnalyzerDemo.MyTestCase1">
    <skipped/>
  </testcase> <!-- test5 -->
  <testcase name="test5" time="0.004" classname="RetryAnalyzerDemo.MyTestCase1">
    <skipped/>
  </testcase> <!-- test5 -->
  <testcase name="test5" time="0.002" classname="RetryAnalyzerDemo.MyTestCase1">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at RetryAnalyzerDemo.MyTestCase1.test5(MyTestCase1.java:34)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
</testsuite> <!-- AnnotationTransformer -->
